promregistry/registry.go

```diff
@@ -0,0 +1,33 @@
+package promregistry
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// TeeRegisterer supports MultipleRegisterer.
+type TeeRegisterer struct {
+	Regs []prometheus.Registerer
+}
```

Comment:
I think we can simplify this even more by defining 
```Go
type TeeRegisterer []prometheus.Registerer
```

And then adjust the methods below a tiny bit.

Reply:
Yes, I should used type alias in the first place. Thanks.